programs/multisig/Cargo.toml

```diff
@@ -16,4 +16,4 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = "0.18.2"
+anchor-lang = "0.19.0"
```

Comment:
Please update npm package in `package.json` / `package-lock.json` and version in `.travis.yml`.